### ISSUE TITLE
Update README with sample code

### ISSUE THREAD

**Issue by NakedFerret** [refinery29/chassis#31]
I'd like some sample code that shows how the framework is used

**Comment by mattmillr** [comment]
Absolutely. That's one of our requirements for release 1.0.0... not only some sample code but a whole "quick start" tutorial modeled after https://docs.djangoproject.com/en/1.8/intro/tutorial01/ (ours will be much simpler)

**Comment by NakedFerret** [comment]
Cool!
